<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Video</class>
 <widget class="QDialog" name="Video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>VÃ¤lj</string>
   </property>
  </widget>
  <widget class="QComboBox" name="topExerciseComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="subExcersiceComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="exerciseTreeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>291</width>
     <height>381</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonImportVideoHelp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Import video data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
